.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "Rust Web Server Development Environment",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	
+  "image": "mcr.microsoft.com/devcontainers/rust:1-1-bookworm",
+	"forwardPorts": [
+    5000
+  ],
+  "customizations": {
+		"vscode": {
+			"extensions": [
+				"rust-lang.rust-analyzer",
+				"tamasfe.even-better-toml",
+				"serayuzgur.crates"
+			]
+		}
+	},
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "cargo run"
```

Comment:
I guess it should be cargo build

Reply:
Yes you are right but with `cargo build` it doesn't run the project only builds it. We can add that too. 👍

Reply:
Yeah, in postCreateCommand we should add command for building only.